.github/workflows/codecov.yml

```diff
@@ -6,29 +6,27 @@ env:
 jobs:
   tarpaulin:
     name: tarpaulin
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+
     - name: Install netcdf
       run: sudo apt-get install libnetcdf-dev
+
     - name: Install rust
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        profile: minimal
-        override: true
+      uses: dtolnay/rust-toolchain@stable
+
     - name: Install tarpaulin
-      uses: actions-rs/install@v0.1
+      uses: baptiste0928/cargo-install@v2.1.0
```

Comment:
Might be a good idea to pin this to a hash, in case it gets compromised -- tags can be replaced easily.

Reply:
Thanks, pinned using the long hash for this action